GNUmakefile

```diff
@@ -19,7 +20,7 @@ build: fmtcheck
 	go build \
 		-tags release \
 		-ldflags '-X $(MODULE)/internal/config.Version=$(VERSION)' \
```

Comment:
Need to update to `VERSION_NUMBER` here too

Reply:
This is for the user-agent and imo it should have the v and the -dirty 😄